README.md

```diff
@@ -21,15 +15,14 @@ Inside of each category you'll find a **README.md** file, which will contain a m
 
 *Happy Integration!*
 
+## Note
+
+This (main) branch requires Spring Integration 5.0 or above. For samples running against earlier versions of Spring Integration, use the __SI4.3.x__ and other branches.
```

Comment:
This has to be up-to-date: `6.0` is the current one. `5.5.x` for the previous supported version.

Reply:
Updated